specs/2025-02-24.yaml

```diff
@@ -1023,137 +938,449 @@ paths:
+        foundation_code:
+          $ref: "#/components/schemas/TaxExemptCode"
+        affiliation_code:
+          $ref: "#/components/schemas/TaxExemptCode"
+        organization_code:
+          $ref: "#/components/schemas/TaxExemptCode"
+        exempt_status_code:
+          $ref: "#/components/schemas/TaxExemptCode"
+        deductibility_code:
+          $ref: "#/components/schemas/TaxExemptCode"
+        ntee_code:
+          $ref: "#/components/schemas/NteeCode"
+        naics_code:
+          $ref: "#/components/schemas/NaicsCode"
+    NonprofitCompliance:
+      type: object
+      description: |-
+        An object that contains legal information about the nonprofit entity and its tax-exemption status.
+        This information is used to comply with regulations and ensure the legitimacy of the nonprofit for purposes of donations and disbursements.
+      properties:
+        daf_eligible:
           type: boolean
-          readOnly: true
-          description: |-
-            A flag to indicate if the nonprofit is in good standing with the IRS.
-            If the nonprofit is a tax-exempt 501(c)(3) Public Charity in good standing with the IRS, this field should be true.
-            This status can change over time and is kept up-to-date by Chariot.
-            Regardless of the value of this field, Connects can still be created for the nonprofit,
-            however the nonprofit will not be able to receive grants through Chariot if this field is false.
-            If you believe the value of this field is incorrect for a Nonprofit, please contact the Chariot team.
+          description: Whether the entity is eligible to receive donations from a Donor Advised Fund
           example: true
-        claimed:
-          type: boolean
-          readOnly: true
-          description: |-
-            A flag to indicate if the nonprofit has been claimed by a user.
-            A nonprofit is claimed if a user signs up for a Chariot account with
-            this nonprofit and is verified by the Chariot team.
-    Suborganization:
+        irs_pub_78:
+          $ref: "#/components/schemas/ComplianceRequirement"
+        ofac:
+          $ref: "#/components/schemas/ComplianceRequirement"
+        foundation_code:
+          $ref: "#/components/schemas/ComplianceRequirement"
+        california_attorney_general_registry:
+          $ref: "#/components/schemas/ComplianceRequirement"
+        california_franchise_tax_board:
+          $ref: "#/components/schemas/ComplianceRequirement"
+    Person:
       type: object
       description: |-
-        A suborganization represents an organization that is under the umbrella of a parent EIN.
-        This is common for nonprofits that have multiple chapters or locations or operate as a fiscal sponsor.
+        A person is an individual who is an officer of a nonprofit.
       required:
         - id
         - name
-        - ein
+        - email
+        - physical_address
```

Comment:
can these be nullable? we aren't storing email or physical address in our NonprofitOfficer table currently which is what we use to populate this

Reply:
yes